src/TConsole.cpp

```diff
@@ -916,8 +916,10 @@ void TConsole::slot_toggleReplayRecording()
     } else {
         if (!mReplayFile.commit()) {
             qDebug() << "TConsole::slot_toggleReplayRecording: error saving replay: " << mReplayFile.errorString();
+            printSystemMessage(tr("Replay recording has been stopped, but couldn't be saved.").arg(mReplayFile.fileName()) % QChar::LineFeed);
```

Comment:
No positional parameter to take the `.arg(...)` Qt will moan about that at runtime (I know because I made the same mistake myself yesterday! :laughing:) : 
```suggestion
            printSystemMessage(tr("Replay recording has been stopped, but couldn't be saved.") % QChar::LineFeed);
```

Reply:
Thanks, fixed.